types/token.go

```diff
@@ -63,6 +63,46 @@ const (
 	POA       TokenType = "POA"
 )
 
+func GetTokenTypes() []TokenType {
```

Comment:
is there a way to add unit test that will make sure it includes all `TokenType` values in the code?

Reply:
I think it's possible if we would generate this go file with .yml token list (just like coins.yml in coins package)

Reply:
@covain what do you think about it?